### DISCUSSION TITLE
Object fitting options for image fields

### DISCUSSION THREAD

**Discussion by cdanny358** [directus/directus#5219] [Archive]
Image fields by default seem to have `object-fit: cover` enabled, like this:
![cover](https://user-images.githubusercontent.com/390911/115913875-0a182880-a426-11eb-8bcc-6b5d057283be.png)
This is less than optimal for many images. Are there options anywhere to change this to some other mode, like `contain`?
![contain](https://user-images.githubusercontent.com/390911/115913977-2caa4180-a426-11eb-9936-1c5cd24b8cb6.png)

**Comment by benhaynes** [comment]
Seems like a really good option to add to the Image Interface! Care to take a stab at a PR?

**Reply by cdanny358** [reply]
Unfortunately, I don't know a thing about Vue. All my work is in React.

**Comment by rijkvanzanten** [comment]
The first one is arguably more _in your face_ though (i'll see myself out)
